deployment/build_and_release/live/ci/terragrunt.hcl

```diff
@@ -48,5 +48,8 @@ inputs = merge(
 
     # Pinned at tag [v20231018](https://github.com/usbarmory/armory-ums/releases/tag/v20231018)
     armory_ums_version: "850baf54809bd29548d6f817933240043400a4e1"
+
+    rest_distributor_base_url = "https://api.transparency.dev"
+    transparencydev_base_url = "https://api.transparency.dev/armored-witness-firmware"
```

Comment:
Would "firmware_base_url" be tighter?

Separately, since this is in both envs, should we just put it into the module or is it better here?

Reply:
Done, thanks.